README.md

```diff
@@ -76,29 +76,16 @@ restart as well.
 
 You can pull our Docker images from [Docker Hub](https://hub.docker.com/r/timescale/timescaledb/).
 
-```bash
-docker pull timescale/timescaledb:latest
-```
+To run, simply issue the following command:
 
-To run, you'll need to specify a directory where data should be
-stored/mounted from on the host machine. For example, if you want
-to store the data in `/your/data/dir` on the host machine:
 ```bash
-docker run -d \
-  --name timescaledb \
-  -v /your/data/dir:/var/lib/postgresql/data \
-  -p 5432:5432 \
```

Comment:
The docker run instructions are no longer an issue for the README, as this section has been removed in `master`. But still applies to the web documentation.